action.yml

```diff
@@ -0,0 +1,139 @@
+# Copyright 2024 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+name: Setup Zephyr project
+description: Setup a Zephyr base project using west and downloading the Zephyr SDK
+
+inputs:
+  app-path:
+    description: Application code path, should contain a west.yml file
+    required: true
+
+  sdk-version:
+    description: Zephyr SDK version to use
+    required: false
+    default: 0.16.4
+
+  sdk-base:
+    description: Base URL of the Zephyr SDK
+    required: false
+    default: https://github.com/zephyrproject-rtos/sdk-ng/releases/download
+
+  toolchains:
+    description: List of toolchains to install, colon separated
+    required: false
+    default: arm-zephyr-eabi:riscv64-zephyr-elf
```

Comment:
the default feels arbitrary but I don't what would be a more sensible one? just arm?

Reply:
Yeah ARM is the obvious one, I wanted to have two to show how to specify multiple ones but then I guess that should go in the help anyway, it's keep just arm. We could also just have this as required and no defaults but I like to have one to show as an example.